### ISSUE TITLE
Session log 22.3.16

### ISSUE THREAD

**Issue by ympaik87** [medal-contender/nbme-score-clinical-patient-notes#58]
# 회의록 22.03.16

1. Roberta-large training 오래 걸리는 문제: 1fold로도 충분히 테스팅 가능. base모델로 테스팅 -> large 적용 방식 보다 large모델 1fold로 성능 테스팅 하는 방식이 더 낫다는 결론.

2. 재우님 Electra submission때 error 발생. 디버깅 해보기로.

3. Roberta에 hidden state에 dropout 추가하는 방식은 성능 향상이 오르지 않음.

4. k fold 방식 중에서 skf 방식 혹은 StratifiedGroupKFold가 더 적합

5. Training set에 annotion이 일관되지 않은 부분이 많이 발견됨. Dataset의 오류를 수정해야 더 정확한 training이 될 것이라 생각됨.

# 다음주 까지

1. 현훈: data annotation 작업
2. 재우: 고양이 inference코드 train화
3. 정원, 영민: Deberta 모델 Feedback Prize 대회 참고해서 튜닝하기.
